flang/include/flang/Lower/OpenMP/Utils.h

```diff
@@ -50,7 +50,7 @@ using DeclareTargetCapturePair =
               const Fortran::semantics::Symbol &>;
 
 mlir::omp::MapInfoOp
-createMapInfoOp(fir::FirOpBuilder &builder, mlir::Location loc,
+createMapInfoOp(mlir::OpBuilder &builder, mlir::Location loc,
```

Comment:
I'm not against changing this to the more generic mlir::OpBuilder, but I am wondering if we should just keep it the same as most of the other flang utilities to keep it somewhat standardized, but I'll leave that up to you!